plugins/exporter.koplugin/target/memos.lua

```diff
@@ -135,7 +135,7 @@ function MemosExporter:createHighlights(booknotes)
             if clipping.note then
                 highlight = highlight .. clipping.note .. "\n\n"
             end
-            highlight =  highlight .. booknotes.title .. " (page: " .. clipping.page .. "）\n\n #" .. booknotes.title .. " #koreader"
+            highlight =  highlight .. booknotes.title .. " (page: " .. clipping.page .. "）\n\n #" .. booknotes.title:gsub("%s+", "_") .. " #koreader"
```

Comment:
It's fine with e.g. `\/:*?"<>|`? (The traditional filesystem thingsj.)

Reply:
I'm not sure I understand what you exactly mean. Would you prefer having ```\/:*?"<>|``` rather than ```_```
for example creating a tag on memos such as
```#This?is?a?book?title```
?

Reply:
I'm just asking if there are any other problematic characters.